modules/installation-initializing-manual.adoc

```diff
@@ -51,27 +54,27 @@ endif::[]
 [id="installation-initializing-manual_{context}"]
 = Manually creating the installation configuration file
 
-ifndef::aws-china,aws-gov,azure-gov,ash,aws-private,azure-private,gcp-private[]
+ifndef::aws-china,aws-gov,aws-secret,ash,aws-private,azure-private,gcp-private[]
 For user-provisioned installations of {product-title}, you manually generate your installation configuration file.
-endif::aws-china,aws-gov,azure-gov,ash,aws-private,azure-private,gcp-private[]
-ifdef::aws-china,aws-gov[]
+endif::aws-china,aws-gov,aws-secret,ash,aws-private,azure-private,gcp-private[]
+ifdef::aws-china,aws-secret[]
 When installing {product-title} on Amazon Web Services (AWS) into a region
 requiring a custom {op-system-first} AMI, you must manually generate your
 installation configuration file.
-endif::aws-china,aws-gov[]
+endif::aws-china,aws-secret[]
```

Comment:
@staebler That is an interesting point. I would like to take it a step further. In the interest of brevity, I do not believe we to need give a reason _why_  you have to create install-config.yaml manually, so much so as it is simply a requirement of the partition. For example, Installing the cluster requires that you manually generate the installation configuration file.